compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -131,6 +131,7 @@
     public final BType typeDesc = new BTypedescType(this.anyType, null);
 
     public final BType semanticError = new BType(TypeTags.SEMANTIC_ERROR, null);
+    public final BType emptyTypeSet = new BType(TypeTags.EMPTY_TYPE_SET, null);
```

Comment:
Should we call this and other related variables/constants `emptyType` instead?

This is what we'll be replacing with the `never` type anyway right?

Reply:
Wouldn't that give the idea that it's a type with no values? This type is used for indicating an empty set of type descriptors. How about `nullSet`? 

Yeah, this is a temporary thing. Hoping to replace this with `never`